Cargo.toml

```diff
@@ -35,6 +35,7 @@ global-context = ["std"]
 # if you are doing a no-std build, then this feature does nothing
 # and is not necessary.)
 global-context-less-secure = ["global-context"]
+zeroizable = ["zeroize", "secp256k1-sys/zeroizable"]
```

Comment:
Features should be named the same as crates. You can rename the crate if you need to activate additional features. (see how it's done for `serde` in `bitcoin`)

Reply:
Hmm. Unfortunately, renaming the crate doesn't seem to work here: it seems like the renaming breaks the derive macro. (I could easily be doing something wrong, though!)

I'll leave this open for now. If the direction of the conversation changes and it starts to look like this PR has a chance at being merged I can revisit.

Reply:
If `zeroize` doesn't support an attribute similar to `serde(crate = "...")` then I suggest to open an issue there first.